### PR TITLE
made the default for @@workload to OLTP 

### PR DESCRIPTION
**Pull request by Akilan1999**
## Description
made the default for @@workload to OLTP 

## Related Issue(s)
https://github.com/vitessio/vitess/issues/6171

## Checklist
- [ x] Tests were added or are not required
- [ Not required] Documentation was added or is not required

### CODE REVIEW

**Review by systay — APPROVED** [review]

File: go/vt/vtgate/plugin_mysql_server_test.go [hunk 1]
```diff
@@ -234,7 +234,7 @@ func newTestAuthServerStatic() *mysql.AuthServerStatic {
 func TestDefaultWorkloadEmpty(t *testing.T) {
 	vh := &vtgateHandler{}
 	sess := vh.session(&mysql.Conn{})
-	if sess.Options.Workload != querypb.ExecuteOptions_UNSPECIFIED {
+	if sess.Options.Workload != querypb.ExecuteOptions_OLTP {
 		t.Fatalf("Expected default workload UNSPECIFIED")
```
Comment (harshit-gangal) [inline go/vt/vtgate/plugin_mysql_server_test.go:238]: this error message should be changes to `Expected default workload OLTP`

File: go/vt/vtgate/planbuilder/show.go [hunk 2]
```diff
@@ -167,10 +167,11 @@ func buildShowTableStatusPlan(show *sqlparser.ShowTableStatus, vschema ContextVS
 	// Remove Database Name from the query.
 	show.DatabaseName = ""
 
+	query := sqlparser.String(show)
 	return &engine.Send{
 		Keyspace:          keyspace,
 		TargetDestination: destination,
-		Query:             sqlparser.String(show),
+		Query:             query,
```
Comment (harshit-gangal) [inline go/vt/vtgate/planbuilder/show.go:174]: looks like this was change got added to PR which is not related to workload changes.

**Review by harshit-gangal — COMMENTED** [review]
there is an empty file present `go/test/endtoend/vtgate/vtroot_6701/topo_6702/topo-stderr.txt`

File: go/vt/vtgate/planbuilder/show.go (hunk 2 above)
Comment (Akilan1999) [reply go/vt/vtgate/planbuilder/show.go:174]: This was something GoLand recommeded to do. Do you want me to revert it back as it was before.

File: go/vt/vtgate/plugin_mysql_server_test.go (hunk 1 above)
Comment (Akilan1999) [reply go/vt/vtgate/plugin_mysql_server_test.go]: Done

**Pull request merged**